packages/utils/lib/environment/parse.ts

```diff
@@ -99,7 +99,7 @@ export const ENVS = z.object({
     FLEET_TIMEOUT_FINISHING_MS: z.coerce
         .number()
         .optional()
-        .default(24 * 60 * 60 * 1000), // 24 hours
+        .default(25 * 60 * 60 * 1000), // 25 hours
```

Comment:
just to be 100% sure there is absolutely no chance the timeout can overlap with a very long sync